README.md

```diff
@@ -1,13 +1,27 @@
 # enclave-cc
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fconfidential-containers%2Fenclave-cc.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fconfidential-containers%2Fenclave-cc?ref=badge_shield)
+## Introduction
 
+  Confidential containers are the product of the combination of confidential computing technology and cloud-native technology. Confidential containers are hardware-based (HW-TEE) for resource isolation, data encryption, and remote attestation, and can prevent theft and tampering of data in the execution environment by cloud service providers and any high-privileged third parties, thus effectively securing user data and assets.
```

Comment:
```suggestion
  Confidential containers are the product of the combination of confidential computing technology and cloud-native technology. Confidential containers use hardware-based Trusted Execution Environments (HW-TEE) for resource isolation, data encryption, and remote attestation. They can protect data in execution from cloud service providers and other high-privileged third parties.
```

Reply:
some of wording changes IMO:
data encryption -> data protection
high-privileged -> privileged